digdag-standards/src/main/java/io/digdag/standards/operator/HttpOperatorFactory.java

```diff
@@ -177,13 +180,36 @@ else if (user.isPresent()) {
                 request.header(header.getKey(), header.getValue());
             }
 
+            configureQueryParameters(request);
+
             ContentResponse response = PollingRetryExecutor.pollingRetryExecutor(state, "request")
                     .withErrorMessage("HTTP request failed")
                     .run(s -> execute(request));
 
             return result(response, storeContent);
         }
 
+        private void configureQueryParameters(Request request)
+        {
+            // TODO: allow secret query parameters
+
+            try {
+                Config queryParameters = params.parseNestedOrGetEmpty("query");
```

Comment:
The URI rfc calls it `query`. https://tools.ietf.org/html/rfc3986#section-3

Is the name `query` confusing in the below snippet?

```
+fetch:
  http>: https://foo.example.com/api/baz
  query:
    foo: bar
    baz: quux
```

Is this better?

```
+fetch:
  http>: https://foo.example.com/api/baz
  query_string:
    foo: bar
    baz: quux
```